packages/grafana-flamegraph/src/FlameGraph/rendering.ts

```diff
@@ -102,39 +104,77 @@ type RenderFunc = (
   width: number,
   height: number,
   label: string,
-  // Collapsed means the width is too small to show the label, and we group collapsed siblings together.
-  collapsed: boolean
+  // muted means the width is too small, and we just show gray rectangle.
```

Comment:
those are 2 different things so they should have different names. Before we named collapsing when we had small items next to each other and instead of rendering them individually we rendered them as single rectangle. At that time collapsing made sense as a name for that. We don't do that anymore so only difference to those items is that they have different color which we call muted color. So now it makes sense to call those items muted, while items which are collapsed vertically into single item are named collapsed.